src/mongocxx/options/find.cpp

```diff
@@ -211,6 +214,179 @@ const stdx::optional<class read_preference>& find::read_preference() const {
     return _read_preference;
 }
 
+find& find::convert_all_modifiers() {
+    if (!_modifiers) {
+        return *this;
```

Comment:
If we change API to return a copy, I think we need have to return a copy here, too, for consistenecy.